src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterKeyValuePair.cs

```diff
@@ -11,8 +11,11 @@ internal sealed class JsonKeyValuePairConverter<TKey, TValue> : JsonConverter<Ke
         private const string KeyName = "Key";
         private const string ValueName = "Value";
 
-        private static readonly JsonEncodedText _keyName = JsonEncodedText.Encode(KeyName);
-        private static readonly JsonEncodedText _valueName = JsonEncodedText.Encode(ValueName);
+        // "encoder: null" is used since the literal values of "Key" and "Value" should not normally be escaped
```

Comment:
Need to verify that this assumption is OK; without this assumption performance would suffer since the values could no longer be cached (at least can't be cached like below).